web/wp-content/themes/dctx-v2/template-parts/scaffolding/scaffolding-buttons.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-pn9iv3.php, but found phpcs-scan-Pn9iV3.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).